Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar). Note: The `div` wrapper is unnecessary.
diff --git a/output/Checkbox Group with Accessible Description/4/b907b484-ae87-4454-b15b-21a21c9b739f.html b/output/Checkbox Group with Accessible Description/4/b907b484-ae87-4454-b15b-21a21c9b739f.html
index a56624b..f6bbf47 100644
--- a/output/Checkbox Group with Accessible Description/4/b907b484-ae87-4454-b15b-21a21c9b739f.html	
+++ b/output/Checkbox Group with Accessible Description/4/b907b484-ae87-4454-b15b-21a21c9b739f.html	
@@ -1,7 +1,16 @@
 <fieldset aria-labelledby="fruit-label" aria-describedby="fruit-desc">
   <legend id="fruit-label">What fruits do you like?</legend>
   <p id="fruit-desc">You can change this later</p>
-  <label><input type="checkbox" name="fruits" value="bananas"> Bananas</label>
-  <label><input type="checkbox" name="fruits" value="oranges"> Oranges</label>
-  <label><input type="checkbox" name="fruits" value="apples"> Apples</label>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples">
+    Apples
+  </label>
 </fieldset>
\ No newline at end of file
